anycable-go/templates/acme-ingress.yml

```diff
@@ -20,12 +21,23 @@ spec:
     secretName: "{{ template "anycableGo.fullname" $root }}-tls-acme"
   rules:
 {{- range $host := .Values.ingress.acme.hosts }}
+{{- $serviceName := include "anycableGo.fullname" $root }}
   - host: {{ $host | quote }}
     http:
       paths:
-      - path: {{ $httpLocation | quote }}
-        backend:
-          serviceName: "{{ template "anycableGo.fullname" $root }}"
-          servicePort: 80
+        {{- if eq $apiVersion "networking.k8s.io/v1" }}
+        - path: {{ $httpLocation | quote }}
+          pathType: Exact
```

Comment:
I would like to ask just in case: is it really an `Exact` location?

Reply:
I haven't seen any example of using subpaths here ( @Envek and @palkan , please correct me if I'm wrong ).